webapp/portlet/src/main/webapp/WEB-INF/jsp/bannerLogo.jsp

```diff
@@ -71,6 +76,7 @@
   });
   <%} %>
   let params = {
+    space: `<%=spaceObject%>`,
```

Comment:
You need here the space REST object representation and not the space Service Layer DTO. BTW, you may need just the 'isFavorite' attribute only